### ISSUE TITLE
Nameko --version CLI flag

### ISSUE THREAD

**Issue by mths0x5f** [nameko/nameko#646]
**Is your feature request related to a problem? Please describe.**
Yes, because I noticed I can not tell installed Nameko version without dealing with pip. This happened when running the stable release thinking for long that was pre-release

**Comment by mattbennett** [comment]
Thanks for the suggestion @mths0x5f. Would welcome a fix for this.

**Comment by bow** [comment]
@mattbennett I'd be interested in submitting a PR for this :).

Looking at the latest `master`, I see that the version string is stored only in `setup.py`, which I suppose needs to be changed so that it can be exposed via `--version`.

Do you have any preference as to how this is done? I can imagine at least three options:

1. Hard-code the version string somewhere in `nameko/__init__.py` and keep the one in `setup.py`. This is my least preferred since there will be two version strings that can be out of sync.

2. Hard-code the version string somewhere in `nameko/__init__.py` and have `setup.py` import `nameko`. This option has less duplications, but it does limit what can be put in `nameko/__init__.py`, since any third-party imports there will break `setup.py`.

3. Use [versioneer](https://github.com/warner/python-versioneer). This seems the most flexible, as it allows the version to follow the actual git tags. The version string would also reflect the state between official releases by pointing to actual commit hashes. The downside is that [versioneer is another dependency that has its own limitations](https://github.com/warner/python-versioneer/issues). It also introduces at least one auto-generated `.py` file.

**Comment by mattbennett** [comment]
Hi @bow, that'd be great, thanks. Are there downsides of simply using `pkg_resources` for this?

```python
import pkg_resources
print(pkg_resources.get_distribution("nameko").version)
```

**Comment by bow** [comment]
@mattbennett Ah, that is also an option indeed. Given the current setup of the library, it seems that there is little to no downsides. I've opened #662 for this.

**Comment by mattbennett** [comment]
Implemented in #662. Thanks @bow!
